README.md

```diff
@@ -79,6 +79,12 @@ To hourly update documents:
 npm run start:scheduler
 ```
 
+To get the latest versions of a specific service's terms:
```

Comment:
I'd say it's either _versions_ (plural) of a service _documents_ or _version_ (singular) of its _terms_ 🙂 

```suggestion
To get the latest version of a specific service's terms:
```